search-app/src/cmr/search/data/metadata_retrieval/metadata_transformer.clj

```diff
@@ -124,12 +124,21 @@
 (defmethod transform-with-strategy :umm-lib
   [context concept _ target-formats]
   (let [{concept-mime-type :format, metadata :metadata} concept
-        umm (legacy/parse-concept context concept)]
-    (reduce (fn [translated-map target-format]
-              (assoc translated-map target-format
-                     (legacy/generate-metadata context umm target-format)))
-            {}
-            target-formats)))
+        [t1 umm] (u/time-execution (legacy/parse-concept context concept))
+        [t2 result] (u/time-execution (reduce (fn [translated-map target-format]
+                                                (assoc translated-map target-format
+                                                       (legacy/generate-metadata context umm target-format)))
+                                              {}
+                                              target-formats))]
+    (info "transform-with-strategy umm-lib: "
+          "legacy/parse-concept time: " t1
+          "reduce w/ legacy/generate-metadata time: " t2
+          "concept-mime-type: " concept-mime-type
+          "parent request num-concepts: " (:num-concepts concept)
+          "target-formats: " target-formats
+          "provider: " (:provider-id concept)
+          "metadata length: " (count metadata))
```

Comment:
is this metadata length or number of metadata?
Not sure what we are counting in this metadata collection

Reply:
it's the size of the metadata, one question that has been asked in the past is does metadata size correlate to processing time.